ghost/email-service/lib/email-renderer.js

```diff
@@ -476,6 +476,25 @@ class EmailRenderer {
         return this.#renderTemplate(data);
     }
 
+    /**
+     * Get email preheader text from post model
+     * @param {object} postModel
+     * @returns
+     */
+    #getEmailPreheader(postModel) {
+        let plaintext = postModel.get('plaintext');
+        let customExcerpt = postModel.get('custom_excerpt');
+        if (customExcerpt !== null){
```

Comment:
In the old system, and empty `custom_excerpt`. would get transformed into `${postModel.get('title')} – `, maybe we better change this check to accomplish the same? 

```suggestion
        if (customExcerpt) {
```